.grabl/automation.yml

```diff
@@ -32,7 +32,11 @@ build:
     test:
       image: vaticle-ubuntu-20.04
       command: |
-        bazel test //...
+        bazel run //:typedb-extractor -- dist/typedb-all-linux
+        sudo systemd-run ./dist/typedb-all-linux/typedb server
+        sleep 60
+        bazel build //...
```

Comment:
libskiko has problems when the build step hasn't been executed first as it creates some files in the user's home directory. Running test immediately without building can cause errors.